### PR TITLE
Minor fixes for the examples

### PR DESCRIPTION
**Pull request by tillrohrmann**
The java based food ordering example still seems to be broken for me. Just before reaching the delivered state, we switch back to scheduling delivery. Not sure what is happening there.

### CODE REVIEW

File: end-to-end-applications/java/food-ordering/webui/src/services/sendToRestate.ts [hunk 1]
```diff
@@ -36,8 +36,10 @@ export async function createOrder(orderId: string, order: any) {
 
 export async function getStatus(orderId: string) {
   return await (
-    await axios.post(`${RESTATE_HOST}/OrderStatusService/${orderId}/get`, {
+    await axios.post(`${RESTATE_HOST}/OrderStatusService/${orderId}/get`, null, {
       headers: challengeHeaders(),
+      // not sure why this removes the content-type header
+      transformRequest: (data, headers) => {},
```
Comment (tillrohrmann) [inline end-to-end-applications/java/food-ordering/webui/src/services/sendToRestate.ts:42]: W/o this, Axios always adds the `content-type` header 🤷

Comment (slinkydeveloper) [reply end-to-end-applications/java/food-ordering/webui/src/services/sendToRestate.ts:42]: admittedly, a POST request without content-type header is a bit weird, but that's an interesting discovery. Perhaps for getStatus use the `GET` verb...

Comment (slinkydeveloper) [reply end-to-end-applications/java/food-ordering/webui/src/services/sendToRestate.ts:42]: We should probably update this code to use the ingress client, as shown here https://github.com/restatedev/sdk-typescript/blob/main/packages/restate-sdk-clients/src/ingress.ts

**Review by gvdongen — APPROVED** [review]
Thanks for fixing!

File: end-to-end-applications/java/food-ordering/webui/src/services/sendToRestate.ts (hunk 1 above)
Comment (tillrohrmann) [reply end-to-end-applications/java/food-ordering/webui/src/services/sendToRestate.ts:42]: Will leave this as a follow-up to replace the existing code with the typed clients.

Comment (tillrohrmann) [reply end-to-end-applications/java/food-ordering/webui/src/services/sendToRestate.ts]: Replace post with get.

**Comment by tillrohrmann** [comment]
I think the problem with the Java food ordering example seemingly to run multiple times is connected to either producing or consuming from Kafka.

**Pull request merged**